src/App.tsx

```diff
@@ -1,9 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { Routes, Route, Navigate } from 'react-router-dom';
 import './App.scss';
+import 'bulma/css/bulma.min.css';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+import { Header } from './Components/Header';
+import { PeopleTable } from './Components/PeopleTable';
+import { getPeople } from './api/api';
+import { Person } from './Types/Person';
 
-export default App;
+export const App: React.FC = () => {
+  const [people, setPeople] = useState<Person[] | null>([]);
+
+  useEffect(() => {
+    getPeople()
+      .then(setPeople);
+  }, []);
```

Comment:
Fetch people from [API](https://mate-academy.github.io/react_people-table/api/people.json) when PeoplePage is opened, so move this logic to `PeopleTable`